firmware/arduino/chessboard/chessboard.ino

```diff
@@ -63,6 +63,12 @@ int currentX, currentY;
 #define MS2_PIN 3
 #define ENDSTOP_PIN 4
 
+// Return definitions
```

Comment:
Error codes

Reply:
For consistency's sake, what do you think of "Movement function status codes" rather than "Error codes" on and making this an enum instead of preprocessors (similar to how Chase set his stuff up, see [this](https://github.com/ucfai/knightros-gambit/blob/dec0e95f9112a3872012e5101d4dd0ad6e2f45e3/firmware/arduino/chessboard/circleFunction.ino#L5) or [this](https://github.com/ucfai/knightros-gambit/blob/dec0e95f9112a3872012e5101d4dd0ad6e2f45e3/firmware/arduino/chessboard/uart.ino#L10)).

Reply:
PS: I don't think it matters in this case since we have multiple codes for error, but just a note that `0` is typically used for `false`/failure.